src/draw.jl

```diff
@@ -149,16 +149,41 @@ end
 function spannable_xy_labels(layout)
     labs = map(layout.content) do _ax
         ax = _ax.content
-        (x = ax.xlabel[], y = ax.ylabel[], empty = isemptyax(ax))
+        (x = ax.xlabel[], y = ax.ylabel[], 
```

Comment:
This doesn't really work for observables, because once you do `ax.xlabel[]` (note the square brackets), it is no longer responsive. You should take `ax.xlabel` as is, and pass it around (lifting when necessary). Same for `ax.xlabelpadding` (especially because afterwards you wrap it in a new `Node`, which you shouldn't need to). So basically, instead of

```julia
padx = Node(ax.xlabelpadding[]) # no longer responds to changes in `ax.xlabelpadding`
```

you simply want

```julia
padx = ax.xlabelpadding
```